src/NUnit.Portable.Agent/TestPackage.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Reflection;
+
+namespace NUnit.Engine
+{
+    internal class TestPackage
+    {
+        internal Assembly TestAssembly { get; }
+        internal Assembly FrameworkAssembly { get; }
+        internal object FrameworkController { get; }
+
+        private Type FrameworkControllerType => FrameworkController.GetType();
+        public string FullName => TestAssembly.FullName;
+
+        internal TestPackage(Assembly testAssembly, Assembly frameworkAssembly, object frameworkController)
```

Comment:
TestPackage is a fundamental class in the engine API, representing a test file and a number of associated settings.

Changing it to mean something else in this agent doesn't seem like such a good idea, particularly as we try to get netstandard assemblies to run under the engine. There are a wide range of meaningful names available at relatively low cost. :-)